include/mbedtls/ssl.h

```diff
@@ -90,6 +90,7 @@
 #define MBEDTLS_ERR_SSL_PEER_VERIFY_FAILED                -0x7800  /**< Verification of our peer failed. */
 #define MBEDTLS_ERR_SSL_PEER_CLOSE_NOTIFY                 -0x7880  /**< The peer notified us that the connection is going to be closed. */
 #define MBEDTLS_ERR_SSL_BAD_HS_CLIENT_HELLO               -0x7900  /**< Processing of the ClientHello handshake message failed. */
+#define MBEDTLS_ERR_SSL_BAD_HS_CLIENT_HELLO_CCS           -0x7910  /**< Processing of the ClientHello handshake message failed; CCS received instead. */
```

Comment:
The error should be registered in `error.h`, and we should make sure that it isn't taken; the script `scripts/generate_errors.pl` will do that. If you have already registered various other TLS 1.3 specific error codes, though, we might as well leave it for a later point. In this case, I'll create an issue. WDYT?

Reply:
I have added the code to error.h.